fuzz.go

```diff
@@ -503,31 +504,65 @@ type int63nPicker interface {
 	Int63n(int64) int64
 }
 
-type charRange struct {
-	first, last rune
+// UnicodeRange is a structure that provides the developer with setting
+// one encoding range.
+type UnicodeRange struct {
+	First, Last rune
 }
 
+// UnicodeRanges is a slice that provides the developer with setting
```

Comment:
I suggest: UnicodeRanges describes an arbitrary number of sequential ranges of unicode characters. To be useful, each range must have at least one character (First <= Last) and there must be at least one range.

Reply:
Fix